x-pack/legacy/plugins/monitoring/public/lib/setup_mode.js

```diff
@@ -86,18 +86,21 @@ export const updateSetupModeData = async (uuid, fetchWithoutClusterUuid = false)
   const isCloud = chrome.getInjected('isOnCloud');
   const hasPermissions = get(data, '_meta.hasPermissions', false);
   if (isCloud || !hasPermissions) {
-    const text = !hasPermissions
-      ? i18n.translate('xpack.monitoring.setupMode.notAvailablePermissions', {
-        defaultMessage: 'You do not have the necessary permissions to do this.'
-      })
-      : i18n.translate('xpack.monitoring.setupMode.notAvailableCloud', {
-        defaultMessage: 'This feature is not available on cloud.'
+    let text = null;
```

Comment:
Should we maybe set a default `text` here? I guess it can be empty, since the title will still be `Setup mode is not available`